### PR TITLE
[Fika][cli] checking out to recent feature branch

### PR DESCRIPTION
**Pull request by tuxshido**
Notion 다큐먼트: https://www.notion.so/fikadev/Fika-cli-checking-out-to-recent-feature-branch-83c052d11c044cedbcbc5d8f89819216
 해결이슈: #336

### CODE REVIEW

File: src/command/checkout-feature-branch/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Command } from "commander";
+import { asyncWrapper, commandWrapper } from "../wrapper/wrappers";
+import { checkoutFeatureBranchAction } from "./checkout-feature-branch.action";
+
+export const finishCommand = new Command()
+  .command("finish")
```
Comment (croquies) [inline src/command/checkout-feature-branch/index.ts]: Here, also the command is "finish" 
and the name of function is "finishCommand"

**Review by croquies — CHANGES_REQUESTED** [review]

File: src/command/checkout-feature-branch/index.ts (hunk 1 above)
Comment (tuxshido) [reply src/command/checkout-feature-branch/index.ts]: Screen Shot 2022-08-24 at 9 54 12 AM

Comment (tuxshido) [reply src/command/checkout-feature-branch/index.ts]: I think I made an additionnal commit for that, If it is not correct or not visible, I can "fika finish"again

Comment (croquies) [reply src/command/checkout-feature-branch/index.ts]: OK, I found that! you don't need to fika finish again

**Pull request merged**